Resources/Prototypes/_Pirate/Entities/Structures/Doors/Airlocks/access.yml

```
@@ -0,0 +1,120 @@
+    containers:
+      board: [ DoorElectronicsIPCommand ]
+      
+- type: entity
+  parent: AirlockExternalGlassShuttleSyndicateLocked
+  id: AirlockExternalGlassShuttleInterdyneLocked
+  suffix: Interdyne, Locked, Glass
+  components:
+  - type: ContainerFill
+    containers:
+      board: [ DoorElectronicsIP ]
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)